android/build.gradle

```diff
@@ -16,6 +16,6 @@ android {
 }
 
 dependencies {
-    compile 'com.usabilla.sdk:ubform:5.2.+'
-    compile "com.facebook.react:react-native:0.+"
+    implementation 'com.usabilla.sdk:ubform:6.+'
```

Comment:
Do you think we should change this to import the project instead of changing the number everytime?